locales/de.json

```
@@ -53,7 +53,7 @@
 	"View": "Wiedergabe",
 	"Views": "Wiedergaben",
 	"Subscribe": "Abonnieren",
-	"Unsubscribe": "Abmelden",
+	"Unsubscribe": "Kündigen",
```

I'd say "Deabonieren". This is not contract or something similar

---

True, "kündigen" sounds more like cancelling a contract. Then I would suggest to use "Abo beenden". This is also what is used on YouTube.

```suggestion
	"Unsubscribe": "Abo beenden",
```

---

Sounds good 👍